bigtable/admin/table_admin.cc

```diff
@@ -95,6 +95,36 @@ std::vector<::google::bigtable::admin::v2::Table> TableAdmin::ListTables(
   return result;
 }
 
+::google::bigtable::admin::v2::Table TableAdmin::GetTable(
+    std::string table_id, ::google::bigtable::admin::v2::Table::View view) {
+  // Copy the policies in effect for the operation.
+  auto rpc_policy = rpc_retry_policy_->clone();
+  auto backoff_policy = rpc_backoff_policy_->clone();
+
+  btproto::GetTableRequest request;
+  request.set_name(instance_name() + "/tables/" + table_id);
```

Comment:
Performance doesn't matter here but should we make `absl::StrCat` the standard way we do all string concatenation operations?

Reply:
Done.